modules/nw-networkpolicy-about.adoc

```diff
@@ -42,7 +42,12 @@ spec:
 
 * Only allow connections from the {product-title} Ingress Controller:
 +
-To make a project allow only connections from the {product-title} Ingress Controller, add the following `NetworkPolicy` object:
+To make a project allow only connections from the {product-title} Ingress Controller, add the following `NetworkPolicy` object.
++
+[IMPORTANT]
+====
+For the OVN-Kubernetes network provider plug-in, when the Ingress Controller is configured to use the `HostNetwork` endpoint publishing strategy, there is no supported way to apply network policy so that ingress traffic is allowed and all other traffic is denied.
```

Comment:
s/ingress traffic/Ingress traffic

`Ingress` is one of those weird words that we always capitalize (like `Operator`). Not sure of the background behind that decision, but that still stands according to our [term glossary](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#ingress).

Reply:
That's off the wall, but thank you.

Reply:
@mikemckiernan I took a stroll through the [Kube docs](https://kubernetes.io/docs/concepts/services-networking/network-policies/) describing ingress traffic and they are using a lower case `ingress traffic`. I have always referred to the glossary for guidance, but I think you're right that the term there is recommending uppercase for Ingress resources. I think it's fine to disregard my suggestion. Thanks for double checking that. :+1: